go/compress.go

```diff
@@ -77,10 +77,10 @@ func compressExist(exist *ExistenceProof, lookup *[]*InnerOp, registry map[strin
 		return nil
 	}
 	res := &CompressedExistenceProof{
-		Key:   exist.Key,
-		Value: exist.Value,
-		Leaf:  exist.Leaf,
-		Path:  make([]int32, len(exist.Path)),
+		Key:       exist.Key,
+		ValueHash: exist.ValueHash,
```

Comment:
Okay, so this is just the raw bytes includes in the leaf - we can also use KeyHash here, which would make it consistent.

I mean for IAVL I believe this is a NoHash for Key, so KeyHash = Key, but this would make it symetric